initfiles/examples/embed/couchbase-simple.ecl

```diff
@@ -0,0 +1,112 @@
+IMPORT couchbase;
+/******************************************************************************************
+   This set of example queries are based on the couchbase sample bucket 'travel-sample'
+   The sample data is easily set-up, pelase refer to the couchbase documentation for set-up
+*******************************************************************************************/
+
+server := '127.0.0.1';            //change this to point to your couchbase
+thebucket := 'travel-sample';
+
+namerec := RECORD
+    string name;
+END;
+
+travelrec := RECORD
+    integer id;
+    string name;
+    string callsign;
+    string country;
+    string5 iata;
+    string5 icao;
+    string type;
+END;
+
+georec := RECORD
+    integer alt;
+    real lat;
+    real lon;
+END;
+
+airportrec := RECORD
```

Comment:
travelrec and airportrec have overlap, I'm not sure is it's possible/worthwhile to create a base and inherit